platform/macos/joypad_macos.mm

```diff
@@ -228,7 +228,7 @@ @interface JoypadMacOSObserver ()
 @property(assign, nonatomic) BOOL isObserving;
 @property(assign, nonatomic) BOOL isProcessing;
 @property(strong, nonatomic) NSMutableDictionary<NSNumber *, Joypad *> *connectedJoypads;
```

Comment:
How about this one? I'm not familiar with that code but I see it used as GCController too:
```
217:                    GCController *controller = [self.connectedJoypads objectForKey:key];
```

Reply:
It's in iOS code, it's a bit different and always using `GCController *`.